.github/workflows/test-deploy.yml

```
@@ -1,40 +1,73 @@
-name: Release packages and images
+name: Test & Deploy
 on:
-  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * *
   release:
     types: [published]
+  pull_request:
 jobs:
-  package:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - run: npm ci
+      - run: npm run lint
+  test:
+    environment: acceptance-tests
```

```suggestion
    environment: ${{ (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository) && 'test-external' || 'test-internal' }}
```

This should allow us to automatically authorize every internal trigger while retaining the manual confirmation requests for external contributors.